### PR TITLE
[admin] Allows players to view their own notes on-server

### PR DESCRIPTION
**Pull request by Altoids1**
Based on a conversation I had with Mongoose earlier.

There is already a proc for handling this, but it gives the player access to a few bits of information I'd rather they not possess, so I will be snipping off those bits in a mirrored proc.

#### Changelog

:cl:  Altoids
rscadd: Players can now view their own notes on the server!
/:cl:

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
Took you long enough

File: yogstation/code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -41,4 +41,19 @@
...
+	
+	if(check_rights(R_ADMIN,0))// If they're an admin, just give them the actual admin version of this verb.
+		browse_messages(target_ckey = usr.ckey, agegate = FALSE)
+		return
+
+	if(!CONFIG_GET(flag/see_own_notes))
+		to_chat(usr, "<span class='warning'>Sorry, that function is not enabled on this server!</span>")
+		return
+
+
```
Comment (monster860) [inline yogstation/code/modules/client/verbs/ooc.dm]: uh this does like nothing

**Review by monster860 — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (Altoids1) [reply yogstation/code/modules/client/verbs/ooc.dm]: [WORK IN PROGRESS]

I just wanted to roll out a PR a day early to get opinions going beforehand.

**Comment by Oakboscage** [comment]
This defaults all old notes to hidden right?

**Pull request closed without merging**